tests/unit_tests/test_load_forward_model.py

```diff
@@ -69,48 +69,6 @@ def run_simulator(time_step_count, start_date) -> Summary:
     return summary
 
 
-@pytest.mark.usefixtures("copy_snake_oil_case_storage")
-def test_load_inconsistent_time_map_summary(caplog):
-    """
-    Checking that we dont util_abort, we fail the forward model instead
-    """
-    cwd = os.getcwd()
-
-    # Get rid of GEN_DATA as we are only interested in SUMMARY
-    with fileinput.input("snake_oil.ert", inplace=True) as fin:
-        for line in fin:
-            if line.startswith("GEN_DATA"):
-                continue
-            print(line, end="")
-
-    facade = LibresFacade.from_config_file("snake_oil.ert")
-    realisation_number = 0
-    storage = open_storage(facade.enspath, mode="w")
-    ensemble = storage.get_ensemble_by_name("default_0")
-    assert ensemble.get_realization_mask_with_responses()[
-        realisation_number
-    ]  # Check prior state
-
-    # Create a result that is incompatible with the refcase
-    run_path = Path("storage") / "snake_oil" / "runpath" / "realization-0" / "iter-0"
-    os.chdir(run_path)
-    ecl_sum = run_simulator(1, datetime(2000, 1, 1))
-    ecl_sum.fwrite()
-    os.chdir(cwd)
-
-    realizations = [False] * facade.get_ensemble_size()
-    realizations[realisation_number] = True
-    with caplog.at_level(logging.WARNING):
-        loaded = facade.load_from_forward_model(ensemble, realizations, 0)
```

Comment:
This approach plus a manual update is possible, and would be a lot faster than run_cli.

Reply:
Kind of hesitate to do add too touch too much detail in the test.

Reply:
Tried doing this: 
```
def run_update(ert: LibresFacade) -> None:
    n = ert.get_ensemble_size()
    storage = open_storage(ert.enspath, mode="w")
    experiment = storage.create_experiment(
        parameters=list(ert.config.ensemble_config.parameter_configs.values())
    ).id
    prior = storage.create_ensemble(experiment=experiment, ensemble_size=n)
    posterior = storage.create_ensemble(experiment=experiment, ensemble_size=n)

    loaded = ert.load_from_forward_model(prior, np.array([True] * n), 0)
    assert loaded == n
    ert.smoother_update(prior, posterior, "")
```
but there were prior.has_data() fails.

Reply:
I think you need to do create_run_path and also have an ensemble_context. I suggest we just drop it and call run_cli for now.